.travis.yml

```diff
@@ -40,7 +40,7 @@ install:
   - sudo apt-get update
   - sudo apt-get install -y --force-yes oracle-java8-installer ant maven2 --no-install-recommends
   - sudo update-java-alternatives --set java-8-oracle
-  - pip install -r requirements.txt
+  - pip install -r requirements.txt --no-deps --no-cache -U
```

Comment:
this looks unrelated. In any case, I am +1 for it if for now if it fixes things, though I am not sure what is the opinion of other developers.

Reply:
@capooti yes it is unrelated. Thanks for looking.